lib/engine/game/g_1880/step/buy_sell_par_shares.rb

```diff
@@ -113,6 +113,20 @@ def process_par(action)
 
             process_presidents_percent_choice(Action::Choose.new(@parring[:entity], choice: percent_choices.keys.first))
           end
+
+          def process_buy_shares(action)
+            super
+            extra_cap?(action.entity) if @game.full_cap_event
+          end
+
+          def extra_cap?(corporation)
```

Comment:
? methods are not supposed to mutate

Reply:
Also put the below functionality in the game class and share it with the capitalization event

Reply:
right, good catch, typo. I think it was originally supposed to be a boolean, I switched the method def, but not the name.